craco.config.js

```diff
@@ -6,15 +6,21 @@ const path = require('path')
 module.exports = function() {
   return {
     webpack: {
-      alias: {},
+      alias: {
+        constants: path.resolve(__dirname, 'src/custom/constants')
+      },
       plugins: [],
 
       // https://webpack.js.org/configuration
       configure: webpackConfig => ({
         ...webpackConfig,
         resolve: {
           ...webpackConfig.resolve,
-          modules: [path.resolve(__dirname, 'src/custom'), ...webpackConfig.resolve.modules]
+          modules: [
+            path.resolve(__dirname, 'src/custom'),
+            path.resolve(__dirname, 'src'),
```

Comment:
why do you need this?

Reply:
for correct fallback order
without it would be `src/custom, node_modules, src` so `node_modules` would take precedence
can sometimes be a problem